tests/system/CMakeLists.txt

```diff
@@ -141,6 +141,23 @@ IF(BUILD_LMYENGINE_INTERFACE)
 ENDIF(BUILD_LMYENGINE_INTERFACE)
 
 #
+# H4 starting from converged wave function, doing a quick adaptive linear method run (using the same
+# spo set for both spin-up and spin-down determinants) and then doing a DMC calculation
+#
+  LIST(APPEND H4_ORB_OPT_DMC_SCALARS "totenergy" "-2.191461 0.0048") # total energy
+
+IF(BUILD_LMYENGINE_INTERFACE)
+  QMC_RUN_AND_CHECK(short-H4-orb-opt-dmc
+                    "${CMAKE_SOURCE_DIR}/tests/molecules/H4_orb_opt_dmc"
+                    H4
+                    h4_orb_opt.xml
+                    16 1
```

Comment:
Better to have a test with 16 threads so that this test can run without MPI. i.e. "1 16" here, not "16 1"